docs/streaming-programming-guide.md

```diff
@@ -1948,8 +1948,8 @@ unifiedStream.print();
 {% highlight python %}
 numStreams = 5
 kafkaStreams = [KafkaUtils.createStream(...) for _ in range (numStreams)]
-unifiedStream = streamingContext.union(kafkaStreams)
-unifiedStream.print()
+unifiedStream = streamingContext.union(*kafkaStreams)
```

Comment:
@srowen right.  The method is: def union(self, *dstreams), so it expects a variable number of dstreams passed in as arguments.  If you pass in a list of dstreams without unpacking it, then vargs= [kafkaStreams, ] as opposed to vargs = [stream1, stream2, stream3, ...].  In the method then, len(vargs) == 1, so it just returns [kafkaStreams] as the unifiedStream without actually performing the union.  And then unifiedStream.pprint() will fail because unifiedStream is not a dstream but a list.